valve-monitor/firmware/README.md

```diff
@@ -1,41 +1,107 @@
-# Flow Rate Sensor and Valve Controller
+# Valve Monitor Firmware
 
-This directory contains Arduino-based firmware for a flow rate measurement
-utility with optional valve control support.
+The Valve Monitor project’s firmware is built with [Platform.io](https://platformio.org/) 
+as an Arduino application. To run the firmware, start by ensuring you have 
+the prerequisites below installed.
 
-## Requirements
+## Prerequisites
 
-- Visual Studio Code with the [PlatformIO IDE](https://platformio.org/platformio-ide)
-installed.
-- [Notecard](https://blues.io/products/notecard/).
-- [Swan MCU](https://blues.io/products/swan/).
-- [Notecarrier F](https://blues.io/products/notecarrier/notecarrier-f/).
-- Male to male jumper wires.
-- A micro USB data cable.
+1. Download and install [Visual Studio Code](https://code.visualstudio.com/).
+1. Install the [PlatformIO IDE extension](https://marketplace.visualstudio.com/items?itemName=platformio.platformio-ide)
+via the Extensions menu of Visual Studio Code.
+![Visual Studio Code extensions menu with a search of "platformio"](../images/platformio-extension.png)
 
-## Setup
+## Opening
 
-- [Plug the Notecard into the Notecarrier F](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/).
-- Plug the Swan into the Feather-compatible headers on the Notecarrier.
-- Connect the ATTN and F_D13 pins on the Notecarrier.
-- Connect the F_D10 pin of the Notecarrier to your solenoid valve's control pin.
-- Connect the F_D6 pin of the Notecarrier to the signal pin of your flow rate
-meter.
-- Connect the Swan to your development machine with the micro USB cable.
-- Open Visual Studio Code and navigate to the PlatformIO menu using the icon
-on the leftmost side.
-- Select "Open" and open valve-monitor/firmware.
-- Edit main.cpp so to define a valid product UID:
+Once you have Visual Studio Code and the PlatformIO extension installed, you next need
+to open the firmware within PlatformIO.
```

Comment:
Link to PlatformIO here?

Reply:
I linked to it at the top of this README which I think is probably sufficient.